### PR TITLE
Compatibility with Plone 5.2 and Python 3

### LINKED ISSUE #8: Plone 5.2 / Python 3 compatibility
- [x] Migrate collective.mailchimp to work with Python 3 / Plone 5.2
- [ ] Migrate pkg to Plone 5.2
- [ ] Migrate pkg to Python 3

### PR DESCRIPTION
**Pull request by rodfersou**
Fixes #8

### CODE REVIEW

**Comment by tisto** [comment]
@rodfersou changelog entry pls.

**Comment by tisto** [comment]
@rodfersou this will break Plone 4.x compatibility.

**Comment by sneridagh** [comment]
@rodfersou please check how to provide the Plone4 compat that @tisto is asking. Take a look at the `services` in p.restapi. I remember that we did provided a way to maintain both.

**Comment by tisto** [comment]
@rodfersou would you mind adding a changelog entry for your fix?

**Comment by rodfersou** [comment]
@tisto added changelog, can you please guide me on how to make it works on both Plone 4 and Plone 5.2?

File: src/kitconcept/dsgvo/events.py [hunk 1]
```diff
@@ -32,7 +32,8 @@ def user_registered(user, event):
     except CannotGetPortalError:
         return
     now = DateTime()
-    ip = get_ip(user.REQUEST)
+    portal = api.portal.get()
+    ip = get_ip(portal.REQUEST)
```
Comment (tisto) [inline src/kitconcept/dsgvo/events.py:36]: @csenger can you please double check if "get_ip(portal.REQUEST)" returns the correct IP address of the user?

Comment (tisto) [reply src/kitconcept/dsgvo/events.py:36]: @sneridagh can you pls double check if you have time? @rodfersou just said that you recommended him that approach. I am good with this, was just wondering if that really works.

Comment (csenger) [reply src/kitconcept/dsgvo/events.py:36]: @tisto 
* portal.REQUEST works
* In a local, direct access setup get_ip returns the right address
* behind a frontend http server/proxy depends on the configuration.

**Pull request merged**